tests/kernels/ragged_paged_attention_kernel_v3_hd64_test.py

```diff
@@ -176,7 +176,9 @@ def gen_random(shape, dtype):
         )
         output = output[:cu_q_lens[distribution[-1]]]
 
-        dtype_bits = dtypes.bit_width(jnp.dtype(kv_dtype))
+        dtype_bits = (dtypes.bit_width(jnp.dtype(kv_dtype)) if hasattr(
```

Comment:
It seems like this change is quite a lot? Maybe you can move these functionality into some helper function?